### DISCUSSION TITLE
How to open new tab into current location?

### DISCUSSION THREAD

**Discussion by regs01** [microsoft/terminal#17595] [Q&A]
There is an article, but it's not working on new tabs. Working only on duplicating tab.
https://learn.microsoft.com/en-us/windows/terminal/tutorials/new-tab-same-directory

Use parent process directory always opening into c:\windows\system32, no matter of what startup path or active location is.

**Comment by zadjii-msft** [comment]
Could you elaborate more? Could you share your settings? Did you modify your shell prompt? Are you running as admin? What are you launching (for example, something like `ssh` won't be able to give us useful information about the remote machine's CWD)? 

A screenshot is worth 1000 words 😉

**Reply by regs01** [reply]
Video would explain better. Prompt is working, but inly with duplicate tab.

[screen2gif-2024-07-26-1325.webm](https://github.com/user-attachments/assets/38585d02-7af0-441e-a27a-38d10b9c4b00)
